src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -383,11 +388,11 @@ private void handleOnDragDropped(TableRow<BibEntryTableViewModel> row, BibEntryT
                         }
                         case MOVE -> {
                             LOGGER.debug("Mode MOVE"); // alt on win
-                            importHandler.getLinker().moveFilesToFileDirAndAddToEntry(entry, files);
+                            importHandler.getLinker().moveFilesToFileDirAndAddToEntry(entry, files, libraryTab.getIndexingTaskManager());
```

Comment:
IoC here too?

Reply:
Having the IndexingTaskManager in ImportHandler would require a lot of refactoring and passing along the IndexingTaskManager in many files. I would really like to avoid that.

Reply:
Ok, I agree, it's probably better to fix the bug now.

But sooner or later we have to deal with that. 😅